packages/expo-sqlite/src/Storage.ts

```diff
@@ -182,14 +182,14 @@ export class SQLiteStorage {
    * Alias for [`setItemAsync()`](#setitemasynckey-value).
    */
   async setItem(key: string, value: string | SQLiteStorageSetItemUpdateFunction): Promise<void> {
-    this.setItemAsync(key, value);
+    await this.setItemAsync(key, value);
   }
 
   /**
    * Alias for [`removeItemAsync()`](#removeitemasynckey) method.
    */
   async removeItem(key: string): Promise<void> {
```

Comment:
```suggestion
  async removeItem(key: string): Promise<boolean> {
    return this.removeItemAsync(key);
  }
```
Maybe @Kudo can clarify why we don't do this

Reply:
Hi @alanjhughes, this alias are trying to be compatible with Async storage api so we have to have the same return type. Here and for other, AsyncStorage api is void. 

The return of our Async fonction are more complète and usefull so i think it's good for direct usage (like boolean is there is a line removed by delete). But for this, we have to "discard" the return of the Async fonction in thé alias to be compatible

Api def : https://github.com/react-native-async-storage/async-storage/blob/main/packages/api/src/AsyncStorage.ts

Reply:
i think the return type as void is what exactly @rtorrente mentioned.
also it makes sense to me to have `await` inside the removeItem/close functions rather than delegating to the call sites. that makes the db operations to be more deterministic.

Reply:
The Api compatibility makes sense. Not sure about using the `await`. We are doing nothing with the result and error propagation is implicit. I would remove the extra overhead if the return types match but it's fine as is